ddtrace/propagation/http.py

```diff
@@ -107,9 +107,12 @@ def _extract_header_value(possible_header_names, headers, default=None):
 def _hex_id_to_dd_id(hex_id):
     # type: (str) -> int
     """Helper to convert hex ids into Datadog compatible ints
-    If the id is > 64 bit then truncate the trailing 64 bit.
+    If the id is > 64 bit and 128bit trace_id generation is disabled then truncate the trailing 64 bit.
+    Otherwise convert the full trace id into lower case hex values.
     "463ac35c9f6413ad48485a3953bb6124" -> "48485a3953bb6124" -> 5208512171318403364
```

Comment:
```suggestion
    If the id is > 64 bit and 128bit trace_id generation is disabled then truncate the trailing 64 bit.
    "463ac35c9f6413ad48485a3953bb6124" -> "48485a3953bb6124" -> 5208512171318403364
    Otherwise convert the full trace id into lower case hex values.
```